src/applications/disability-benefits/all-claims/pages/uploadPtsdDocuments.js

```diff
@@ -1,33 +1,25 @@
 import { uploadDescription } from '../content/fileUploadDescriptions';
 import { ptsd781NameTitle } from '../content/ptsdClassification';
 import { ancillaryFormUploadUi } from '../utils';
+import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
+
+const { completedFormAttachments } = fullSchema.properties;
 
 export const uiSchema = {
   'ui:title': ptsd781NameTitle,
   'ui:description': uploadDescription,
-  ptsd781: ancillaryFormUploadUi('', 'PTSD 781 form'),
+  ptsd781: ancillaryFormUploadUi('', 'PTSD 781 form', {
+    attachmentId: 'VA Form 21-781 - Statement in Support of Claim for PTSD',
```

Comment:
So I see the 781a is using the attachment code but here we're passing the label for the attachment code instead...
```
  {
    value: 'L229',
    label:
      'VA Form 21-0781a - Statement in Support of Claim for PTSD Secondary to Personal Assault',
  },
  {
    value: 'L228',
    label: 'VA Form 21-0781 - Statement in Support of Claim for PTSD',
  },
```
Is `L228` not valid?

Reply:
It's not in this case because it's a different type. Since this is of type string, it's treated differently by the system.